### DISCUSSION TITLE
Clean Repo history

### DISCUSSION THREAD

**Discussion by pixByteBender** [newren/git-filter-repo#456] [Q&A] [answered]
I have a repo which use for more than 10 year, migrated from VSS to TFS and then git.  I would like to keep history for all existing file but would like to remove all deleted file from history. By using Git filter-repo 'analyze' command, i can find out all deleted files in path-deleted-sizes.txt. I wonder is there a command to remove these files or I need to run multiple command based on the list in path-deleted-sizes.txt? or need to write a little program for that? there are more than 10 thousand files for my case :).

**Comment by newren** [comment, marked as answer]
Stuff those paths into a file, then use `--invert-paths --paths-from-file FILENAME` ?
